src/modules/flow/http-server/http-server.c

```diff
@@ -278,6 +268,70 @@ common_handle_response_cb(struct sol_flow_node *node,
     return r;
 }
 
+static bool
+is_sse_request(enum sol_http_method method, const struct sol_http_params *params)
+{
+    uint16_t i;
+    struct sol_http_param_value *param;
+
+    if (method != SOL_HTTP_METHOD_GET)
+        return false;
+
+    SOL_HTTP_PARAMS_FOREACH_IDX (params, param, i) {
+        if (param->type != SOL_HTTP_PARAM_HEADER)
+            continue;
+
+        if (sol_str_slice_str_eq(param->value.key_value.key, "Accept") &&
+            sol_str_slice_str_contains(param->value.key_value.value, "text/stream"))
```

Comment:
as before we need to check the highest priority accept... more complex than "contains" :-(